### PR TITLE
[Interaction] Convert RoV Chapter 1 Missions

### PR DESCRIPTION
**Pull request by claywar**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

* Converts and adds additional messaging to RoV Chapter 1
* Adds Container:noAction() to Interaction Framework to handle cases where no other return is available

### CODE REVIEW

**Comment by claywar** [comment]
Should be good to go at this point to review.

**Review by Xaver-DaRed — APPROVED** [review]

File: scripts/missions/rov/1_05_The_Beginning.lua [hunk 1]
```diff
@@ -74,7 +74,7 @@ mission.sections =
         [xi.zone.NORG] =
         {
             ['_700']       = mission:event(276):setPriority(1005), -- RoV objectives are highest priority, set higher than progressEvent (1000)
-            ['Comitiolus'] = mission:event(6),
+            ['Comitiolus'] = mission:event(6), -- NPC Chat 12321, not event 6
```
Comment (zach2good) [inline scripts/missions/rov/1_05_The_Beginning.lua]: What does this mean?

File: scripts/zones/Mhaura/npcs/Ekokoko.lua [hunk 2]
```diff
@@ -11,6 +11,8 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
+    -- TODO: This is not a random sequence, but 51 is displayed on first interaction ever,
```
Comment (zach2good) [inline scripts/zones/Mhaura/npcs/Ekokoko.lua:14]: Are you going to do this, or should we make an issue for this?

**Review by zach2good — APPROVED** [review]
Looks good 👍

**Comment by zach2good** [comment]
Needs a rebase, apparently

File: scripts/missions/rov/1_05_The_Beginning.lua (hunk 1 above)
Comment (claywar) [reply scripts/missions/rov/1_05_The_Beginning.lua]: Had the wrong item there and made a note to change it, but totally missed.  Replaced with the message received from caps.

File: scripts/zones/Mhaura/npcs/Ekokoko.lua (hunk 2 above)
Comment (claywar) [reply scripts/zones/Mhaura/npcs/Ekokoko.lua:14]: Raised #1385

**Pull request merged**